src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingSolutionCrawlerRegistrationService.cs

```diff
@@ -288,23 +284,34 @@ internal void WaitUntilCompletion(Workspace workspace)
                     coordinator.GetTestAccessor().WaitUntilCompletion();
                 }
             }
+#endif
         }
 
         internal sealed class UnitTestingRegistration
         {
             public readonly int CorrelationId;
-            public readonly Workspace Workspace;
+            public readonly string WorkspaceKind;
+            public readonly SolutionServices Services;
```

Comment:
the general idea here was to get us off of 'Workspace' and onto the constituent pieces that are needed instead.  These pieces are possible today to get (obviously, since we still have workspaces), but they shoudl also be hopefully not a problem to get even as we heavily refactor oop and get rid of hte RemoteWorkspace type.